src/server.h

```diff
@@ -2716,6 +2721,13 @@ int authRequired(client *c);
 void putClientInPendingWriteQueue(client *c);
 client *createCachedResponseClient(int resp);
 void deleteCachedResponseClient(client *recording_client);
+void setTrustedNetworkFlag(client *c, const struct sockaddr_storage *sa);
+int isUnixNetwork(client *c);
+int checkConnFromTrustedNetwork(client *c);
+int isTrustedNetwork(client *c);
+in_addr_t getIPv4Netmask(in_addr_t ip);
+int checkTrustedIP(in_addr_t ip);
+void valkeySortIP(in_addr_t *IPlist, unsigned int IPcount);
```

Comment:
I think the convention is to use 'Ip' in the function name (camel casing)

```suggestion
int checkTrustedIp(in_addr_t ip);
void valkeySortIp(in_addr_t *ip_list, unsigned int ip_count);
```